types.cc

```diff
@@ -2254,9 +2188,50 @@ sstring abstract_type::quote_json_string(const sstring& s) {
     return json::value_to_quoted_string(s);
 }
 
+static sstring cql3_type_name_impl(const abstract_type& t) {
```

Comment:
This also should be moved to cql3/, not now.

Reply:
This one is harder, since we cache the result. I will mark as resolved since it is for another patch series.